### ISSUE TITLE
Screen Title won't go away

### ISSUE THREAD

**Issue by hxykatycolf** [PyvesB/AdvancedAchievements#316]
Hello ! I have recetly installed Advanced Achievements. It's been running fine but we experienced some issue with the Screen Title yesterday. We are running Spigot 1.12.2 and using the latest version of the plugin. The screen title pops up on the screen after an achievement was finished but it stayed there [http://prntscr.com/ie9cfy] I have restarted the server, restart / reset the plugin and remove the plugin etc, but the screen title is still there. I hope I can get some help with fixing the problem. Thank you so much! Keep the good work! :)

**Comment by PyvesB** [comment]
Hello @hxykatycolf!

That's odd. Did this only happen on one occasion? Advanced Achievements doesn't do much with titles, it just sends a network packet to the client so that a title is temporarily displayed to the player, but does maintain any "title" state and doesn't need to take care of title fading out.

If it only affects a single player, try out these commands:
```
/title PLAYER title [{"text":""}]
/title PLAYER subtitle [{"text":""}]
```

Cheers,

Pyves

**Comment by hxykatycolf** [comment]
Hey Pyves !
Since we are still building the server so there were only my bestfriend and I on it. He broke a block and triggered the Cobble achivement, which led to the screen title "Cobble !" appeared and stayed on the screen for a long time. So after I reset the plugin, we both joined again, the title in the attached picture in my last post shown on his screen and stayed there while I only get the achievement unlock notification message in Chat but not the screen. The screen title finally went away today. But we did not make any changes since I posted my message here. That's pretty odd and I was hoping to find a solution to that or try to figure out what is wrong.
Thank you so much for your time!
Cheers,
Katy
